sdks/java/io/datalake/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Can you re-work these dependencies to match the pattern used for other IOs? See io/google-cloud-platform/build.gradel for an example.

New dependencies themselves are included buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

Reply:
Okay, I'll modify it according to this